ProcessMaker/Console/Commands/Install.php

```diff
@@ -193,6 +197,56 @@ public function handle()
         return true;
     }
 
+    /**
+     * Configure the username, email, password of the admin user
+     *
+     */
+    private function fetchUserInformation()
+    {
+        do {
+            $username = $this->anticipate(__('Enter your username'), ['admin'], 'admin');
```

Comment:
Remove translations. At this point there isn't any languages installed.

Reply:
Translations removed